app/controllers/category_controller.rb

```diff
@@ -7,10 +7,26 @@ def show
     @category = Category.find(params[:id])
   end
 
+  def create
+    @category = Category.new(category_params)
+    if @category.save
+      flash[:info] = "Saved Successfully"
+      redirect_to create_path
+    else
+      flash[:danger] = "Creating failed"
+      render "create"
+    end
+  end
+
   def index
     @category = Category.all
```

Comment:
you can use the following code instead to make use of will_paginate gem
`@category = Category.paginate(page: params[:page], per_page: 10 )`